deepchem/models/dft/nnxc.py

```diff
@@ -0,0 +1,183 @@
+from abc import abstractmethod
+from typing import Union
+import torch
+from dqc.utils.datastruct import ValGrad, SpinParam
+from dqc.api.getxc import get_xc
+
+
+class BaseNNXC(torch.nn.Module):
+    """
+    Base class for the NNLDA and HybridXC classes.
+    """
+
+    @abstractmethod
+    def get_edensityxc(
+            self, densinfo: Union[ValGrad, SpinParam[ValGrad]]) -> torch.Tensor:
+        """
+        This method is used to transform the electron density. The output
+        of this method varies depending on the layer.
+
+        Parameters
+        ----------
+        densinfo: Union[ValGrad, SpinParam[ValGrad]]
+            Density information calculated using DQC utilities.
+        """
+        pass
+
+
+class NNLDA(BaseNNXC):
+    """
+    Neural network xc functional of LDA
```

Comment:
Can you add some more basic context? That is, the basic set up that this is a class relevant for density functional theory, a basic explanation of what an exchange correlation functional is?